<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageLimitsWgtClass</class>
 <widget class="QWidget" name="PageLimitsWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Critical board ratings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
      <property name="horizontalSpacing">
       <number>12</number>
      </property>
      <item row="1" column="1">
       <widget class="QSpinBox" name="LowVoltageOffBox">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Max current (power)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max voltage (power)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="MaxCurrentPwrBox">
        <property name="suffix">
         <string> mA</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="MaxVoltagePwrBox">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Low voltage off</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="lowVoltageBox">
        <property name="text">
         <string>Low voltage protection</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="MaxVoltageUsbBox">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max voltage (usb)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="MinVoltageUsbBox">
        <property name="suffix">
         <string> mV</string>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="MaxCurrentUsbBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> mA</string>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max current (usb)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min voltage (usb)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="tempLbl">
        <property name="text">
         <string>Max temperature</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="MaxTempBox">
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string> °C</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="overheatBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Switch off power driver on overheat (&gt;125 °C)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bridgeBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Switch off power driver on H bridge alert</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="borderMissetBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enter Alarm state when edge misset is detected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="stickyAlarmBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sticky Alarm flags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="usbBreakReconnect">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Reconnect USB on link break</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="WRMalarm">
         <property name="enabled">
           <bool>true</bool>
         </property>
         <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
           </sizepolicy>
         </property>
         <property name="text">
           <string>WRM Alarm</string>
         </property>
       </widget>
      </item>
      <item>
        <widget class="QCheckBox" name="ENGalarm">
          <property name="enabled">
            <bool>true</bool>
          </property>
          <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="text">
            <string>Engine response Alarm</string>
          </property>
        </widget>
      </item>  
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lowVoltageBox</tabstop>
  <tabstop>LowVoltageOffBox</tabstop>
  <tabstop>MaxCurrentPwrBox</tabstop>
  <tabstop>MaxVoltagePwrBox</tabstop>
  <tabstop>overheatBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lowVoltageBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowVoltageBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>LowVoltageOffBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
